nemo_curator/utils/fuzzy_dedup_utils/merge_utils.py

```diff
@@ -15,15 +15,20 @@
 import math
 from operator import getitem
 
+import dask
 import numpy as np
+import pandas as pd
 from dask.base import tokenize
 from dask.dataframe.core import new_dd_object
 from dask.dataframe.shuffle import partitioning_index
 from dask.highlevelgraph import HighLevelGraph
 from dask.utils import M
+from packaging.version import Version
 
 from nemo_curator.utils.fuzzy_dedup_utils.shuffle_utils import rearange_by_column_direct
 
+DASK_GT_2023_12_0 = Version(dask.__version__) > Version("2023.12.0")
+
```

Comment:
nit: For consistency can we move this logic to https://github.com/NVIDIA/NeMo-Curator/blob/main/nemo_curator/_compat.py

Reply:
Thanks, I assumed this kind of "compat" code was living somewhere, but didn't look very hard :)